MaterialLib/MPL/CreateMaterialSpatialDistributionMap.cpp

```diff
@@ -28,7 +28,7 @@ createMaterialSpatialDistributionMap(
 
     if (max_material_id > static_cast<int>(media.size() - 1))
     {
-        OGS_FATAL(
+        WARN(
```

Comment:
Please provide explanation in the commit message.

Reply:
Explanation was added in the commit [4ffd04f](https://github.com/ufz/ogs/pull/2671/commits/4ffd04f7a989526c049f36287c482fef489e0e10)

Reply:
Why isn't this an OGS_FATAL? I would expect that bad things could be happen if something in the input project file is inconsistent with the mesh file.

Reply:
The problem originates from the fact that we use the material IDs to differentiate the 1D line elements assigned to the BHE in a 3D mesh. The simplest case would be, we have one material ID assigned to 3D prism elements for all the soil part. Then we have the 2nd material ID assigned to the one BHE. In this case, our max_material_id  == 1. However, we can only define 1 media in the input files (which is the soil media). This means media.size() = 1. The if clause on line 29 is then true and we will trigger the block. If we keep the OGS_FATAL function, then we will not be able to run BHE simulations with different types of materials (soil layers). I discussed this with Dima and we agreed to change it to WARN.

Reply:
Thanks for explaining the background. :fast_forward: